modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layouts/02_my_evp/page-definition.json

```diff
@@ -3,8 +3,29 @@
 		"pageElements": [
 			{
 				"definition": {
+<<<<<<< HEAD
```

Comment:
There is some conflict resolution text added to this file, please delete them and see if the page is working well